openslides/core/templates/base.html

```diff
@@ -22,69 +21,82 @@
 
-                        <!-- login/logout button -->
-                        <div class="btn-group">
-                            {% if user.is_authenticated %}
-                            <a href="#" data-toggle="dropdown" class="btn dropdown-toggle">
-                                  <i class="icon-user"></i>
-                                  <span class="optional-small">{{ user.username }}</span>
-                                  <span class="caret"></span>
-                            </a>
-                            <ul class="dropdown-menu pull-right">
-                                <li><a href="{% url 'user_settings' %}"><i class="icon-cog"></i> {% trans "Edit profile" %}</a></li>
-                                <li><a href="{% url 'password_change' %}"><i class="icon-lock"></i> {% trans "Change password" %}</a></li>
-                                <li class="divider"></li>
-                                <li><a href="{% url 'user_logout' %}"><i class="icon-off"></i> {% trans "Logout" %}</a></li>
-                            </ul>
-                            {% else %}
-                                <a href="{% url 'user_login' %}" class="btn"><i class="icon-login"></i> {% trans "Login" %}</a>
-                            {% endif %}
-                        </div>
+                <!-- Search field -->
+                <form id="searchform" class="navbar-form navbar-right" action="{% url 'core_search' %}" method="get">
+                    <div class="form-group input-group">
+                        <input type="text" id="id_q" name="q" class="form-control" placeholder="{% trans 'Search' %}">
+                        <span class="input-group-btn">
+                            <button type="submit" class="btn btn-default">
+                                <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
+                            </button>
+                        </span>
                     </div>
-                    {% endblock %}
-                </div> <!--/#header-->
-            </div> <!--/.span12-->
+                </form>
+            {% endblock %}
         </div>
-    </div> <!--/.container-fluid-->
+    </nav>
+
 
     {% block body %}
     <!-- Container -->
     <div class="container-fluid" id="container">
-    <div class="row-fluid">
+    <div class="row">
 
         <!-- Sidebar navigation (main menu) -->
-        <div class="span2 leftmenu lefticon">
+        <div class="col-md-2 leftmenu lefticon">
             <ul>
                 {% for entry in main_menu_entries %}
                     <li{% if entry.is_active %} class="active"{% endif %}>
                         <a href="{{ entry.get_url }}" class="tooltip-right">
-                        <span class="ico"><i class="{{ entry.get_icon_css_class }}"></i></span>
+                        <!--TODO-->
+                        <span class="glyphicon glyphicon-{{ entry.get_icon_css_class }}" aria-hidden="true"></span>
```

Comment:
Can you please replace `glyphicon-{{ entry.get_icon_css_class }}` by `{{ entry.get_icon_css_class }}` and add the `glyphicon-` prefix at the correct place in all main_menu.py files? This is because plugins might not want to use a glyphicon class but customized css classes to add customized icons.